### ISSUE TITLE
make errors

### ISSUE THREAD

**Issue by joeblew99** [tulir293/mauirc#1]
i am using this repo as a best practice for using gopherjs.
SO learning...

go 1.7

Not sure why make it complaining...

```
/Users/apple/workspace/go/src/maunium.net/go/mauirc
x-MacBook-Pro:mauirc apple$ make clean
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
rm -f index.js.map index.js index.html index.css templates.html mauirc.tar.xz

```

**Comment by tulir** [comment]
It seems that the BSD `find` (which macOS uses) is slightly stricter than GNU `find`. GNU doesn't require the path, but BSD does. Should be fixed in fc8caec2f602ac29d56b9b9b16f57526d600c9cd

**Comment by joeblew99** [comment]
that fixed it. thanks
